### PR TITLE
feat: add no-show-to-edit-list

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

Fixes: CAL-4583

Adds a way to mark members as no show in dropdown

image
image

## How should this be tested?
Goto bookings/past
Click edit dropdown
Click mark as no show
Mark users as noshow

### CODE REVIEW

File: apps/web/components/booking/BookingListItem.tsx [hunk 1]
```diff
@@ -999,6 +1008,75 @@ const GroupedAttendees = (groupedAttendeeProps: GroupedAttendeeProps) => {
...
+  );
+
+  const noShowMutation = trpc.viewer.markNoShow.useMutation({
+    onSuccess: async (data) => {
+      const newValue = data.attendees[0];
+      setNoShowAttendees((old) =>
+        old.map((attendee) =>
+          attendee.email === newValue.email ? { ...attendee, noShow: newValue.noShow } : attendee
+        )
+      );
```
Comment (sean-brydon) [inline apps/web/components/booking/BookingListItem.tsx:1040]: Keeping state locally for this modal was easier than re-validating the entire booking list to change the noshow value

File: apps/web/components/booking/BookingListItem.tsx [hunk 2]
```diff
@@ -257,14 +258,14 @@ function BookingListItem(booking: BookingItemProps) {
...
-      label: t("reassign"),
+      id: "no_show",
+      label: t("mark_as_no_show"),
       onClick: () => {
-        setIsOpenReassignDialog(true);
+        setIsNoShowDialogOpen(true);
       },
-      icon: "users" as const,
+      icon: "eye-off" as const,
     });
```
Comment (sean-brydon) [inline apps/web/components/booking/BookingListItem.tsx:269]: Tf happened here - fixing

**Review by PeerRich — DISMISSED** [review]

File: apps/web/components/booking/BookingListItem.tsx [hunk 3]
```diff
@@ -268,6 +269,17 @@ function BookingListItem(booking: BookingItemProps) {
...
   }
 
+  if (!booking.isBookingInPast) {
```
Comment (PeerRich) [inline apps/web/components/booking/BookingListItem.tsx:272]: only for past bookings right?

**Review by emrysal — DISMISSED** [review]
This looks good to me now, strange merge thing.

**Review by emrysal — APPROVED** [review]

**Pull request merged**